packages/renderer/src/lib/welcome/WelcomePage.svelte

```diff
@@ -0,0 +1,83 @@
+<script lang="ts">
+import bgImage from './background.png';
+import DesktopIcon from '../images/DesktopIcon.svelte';
+import Fa from 'svelte-fa/src/fa.svelte';
+import { faCircle } from '@fortawesome/free-solid-svg-icons';
+import { onMount } from 'svelte';
+import { WelcomeUtils } from './welcome-utils';
+
+let showWelcome;
```

Comment:
we probably need a component test of WelcomePage

we have tests for WelcomeUtils but would be nice to test that if we have buttons that can be clicked and that they're doing something (redirecting people)

Reply:
Added component test in second commit.